src/types/user.types.ts

```diff
@@ -1,10 +1,13 @@
-export interface UserTypes {
-    id: number;
-    userName: string;
-    email?: string;
-    password?: string;
-    isVerified?: boolean;
-    isOnboarding?: boolean;
-    createdAt?: string;
-    updatedAt?: string;
+export interface User {
+    message: string;
+    data: {
+        id: number;
+        userName: string;
+        email?: string;
+        password?: string;
+        isVerified?: boolean;
+        isOnboarding?: boolean;
+        createdAt?: string;
+        updatedAt?: string;
```

Comment:
- do we need `createdAt` and `updatedAt` keys here?

Reply:
As it is optional we can neglect it